animation_nodes/nodes/gpencil/offset_gp_layer_frame_number.py

```diff
@@ -0,0 +1,43 @@
+import bpy
+from ... base_types import AnimationNode, VectorizedSocket
+from ... data_structures import VirtualDoubleList, VirtualPyList, GPLayer
+
+class OffsetGPLayerFrameNumberNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_OffsetGPLayerFrameNumberNode"
+    bl_label = "Offset GP Layer Frame Number"
+
+    useLayerList: VectorizedSocket.newProperty()
+    useFrameNumberList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput(VectorizedSocket("GPLayer", "useLayerList",
+            ("Layer", "layer"), ("Layers", "layers")), dataIsModified = True)
+        self.newInput(VectorizedSocket("Float", "useFrameNumberList",
+            ("Offset Frame Number", "offset"), (" Offset Frame Numbers", "offsets")), value = 0)
```

Comment:
Just call this *Offset*.

Reply:
Okay. I have a concern with the name of the node. Do you have any idea?

Reply:
I am not sure. But I think *Offset GP Layer Frames* is ok.

Reply:
Okay.